client/packages/cli/src/cli.ts

```diff
@@ -19,12 +19,13 @@ const program = new Command()
 program
   .name("t3rn CLI")
   .description("CLI for interacting with the t3rn blockchain")
-  .version("0.1.0")
+  .version("0.1.1")
```

Comment:
Is this the version that needs to be updated @3h4x

Reply:
Yeah but 0.1.1 is already released so it will fail.

Reply:
@chexware packages/sdk/package.json version